subdl

```diff
@@ -89,6 +97,29 @@ def file_base(filename):
 def gunzipstr(zs):
     return gzip.open(io.BytesIO(zs)).read()
 
+def filterSubs(filename):
+    import pysrt
+    print("Parsing subs...")
+    subs = pysrt.open(filename)
+    print(" Length: %d" % len(subs))
+    i = 0
+    removed = 0
+    while i < len(subs):
+        sub = subs[i]
+        rem = False
+        for f in FILTERS:
```

Comment:
you can join all FILTERS and pass it as one single pattern.

    f = '|'.join(FILTERS)

thus, you don't have to iterate over FILTERS.